beetsplug/convert.py

```diff
@@ -208,7 +208,7 @@ def encode(self, command, source, dest, pretend=False):
         except OSError as exc:
             raise ui.UserError(
                 u"convert: could invoke '{0}': {1}".format(
```

Comment:
I know you didn't change this but I think `could invoke` should be `couldn't invoke`. Would it be possible for you to fix this too?